internal/model/naming.go

```
@@ -0,0 +1,138 @@
+		if unicode.IsLetter(r) {
+			return r
+		}
+		if unicode.IsDigit(r) {
+			return r
+		}
+
+		return rune('_')
+	}, s)
+	for _, p := range strings.Split(s, "_") {
+		if _, found := conf.InitialismsMap[p]; found {
+			out += strings.ToUpper(p)
```

golangci-lint (2.12.2)

[error] concat-loop: string concatenation in a loop

(perfsprint)